doc/source/overview/faq.md

```diff
@@ -0,0 +1,56 @@
+ - Check that your model signature (it's type and expected inputs/outputs) are in the right format. For further details refer to [White-box and black-box models](../overview/white_box_black_box.md).
+
+**Q. The explanations are taking a long time to complete, what to do?**
+
+**A.** Explanations can take different times as a function of the model, the data and the explanation type itself. Refer to the [Introduction](../overview/high_level.md) and Methods sections for notes on algorithm complexity. You might need to experiment with the type of model, data points (specifically feature cardinality) and method parameters to ascertain if a specific method scales well for your use case.
+
+**Q. The explanation returned doesn't make sense to me, what to do?**
+
+**A.** Explanations reflect the decision-making process of the model and not that of a biased human observer, see [Biases](../overview/high_level.md#biases). Moreover, depending on the method, the data and the model, the explanations returned are valid but may be harder to interpret (e.g. see [Anchor explanations](#anchor-explanations) for some examples).
+
+**Q. Is there a way I can get more information from the library during the explanation generation process?**
+
+**A.** Yes! We use [Python logging](https://docs.python.org/3/howto/logging.html) to log debug messages from the library. You can configure logging for your script to see these messages during the execution of the code. Additionally, some methods also expose a `verbose` argument to print information to standard output.
+
+## Anchor explanations
+
+**Q. Why is my anchor explanation empty (tabular or text data) or black (image data)?**
+
+**A.** This is expected behaviour and signals that there is no salient subset of features that is necessary for the prediction to hold. This can happen in two ways:
+ - The predicted class of the data point does not change regardless of the perturbations applied to it.
+ - The predicted class of the data point always changes regardless of the perturbations applied to it.
+
+**Q. Why is my anchor explanation so long (tabular or text data) or covers much of the image (image data)?**
+
+**A.** This is expected behaviour and can happen in two ways:
+ - The data point to be explained lies near the decision boundary of the classifier. Thus, many more predicates are needed to ensure that a data point keeps the predicted class as small changes to the feature values may push the prediction to another class.
+ - For tabular data, sampling perturbations is done using a training set. If the training set is imbalanced, explaining a minority class data point will result in oversampling perturbed features typical of majority classes so the algorithm would struggle finding a short anchor exceeding the specified precision level. For a concrete example, see [Anchor explanations for income prediction
```

Comment:
@jklaise the latex build is failing due to this typo (the little square at the end of the line is causing an unrecognised unicode character error).